requirements.txt

```diff
@@ -100,3 +88,17 @@ Jinja2==2.6
 
 # OWSLib
 pytz
+
+-e git+https://github.com/GSA/ckan.git@datagov#egg=ckan
+-e git+https://github.com/GSA/ckanext-archiver@datagov#egg=ckanext-archiver
+-e git+https://github.com/GSA/ckanext-datagovtheme@master#egg=ckanext-datagovtheme
+-e git+https://github.com/GSA/ckanext-datajson@datagov#egg=ckanext-datajson
+-e git+https://github.com/GSA/ckanext-extlink@master#egg=ckanext-extlink
+-e git+https://github.com/GSA/ckanext-geodatagov@master#egg=ckanext-geodatagov
+-e git+https://github.com/GSA/ckanext-googleanalyticsbasic@master#egg=ckanext-googleanalyticsbasic
+-e git+https://github.com/GSA/ckanext-harvest@datagov#egg=ckanext-harvest
+-e git+https://github.com/GSA/ckanext-saml2@bionic-m2crypto#egg=ckanext-saml2
+-e git+https://github.com/GSA/ckanext-qa@datagov#egg=ckanext-qa
+-e git+https://github.com/GSA/ckanext-report@datagov#egg=ckanext-report
+-e git+https://github.com/GSA/ckanext-spatial@datagov#egg=ckanext-spatial
+-e git+https://github.com/GSA/ckanext-ga-report@datagov#egg=ckanext-ga-report
```

Comment:
I know there's a reason these are at the bottom, but now I forget why. Would you include a comment?

Reply:
Here's [the comment I made on this change for `inventory-app`](https://github.com/GSA/inventory-app/pull/18/files#diff-b4ef698db8ca845e5845c4618278f29aR74-R76).

Reply:
I added Brent's comment.